app/views/layouts/_loading_page.html.erb

```diff
@@ -0,0 +1,34 @@
+<body onload="myFunction()" style="margin:0;">
+<div id="loader-wrapper">
+    <div id="loader"></div>
+
+     <div class="loader-section section-left"></div>
+     <div class="loader-section section-right"></div>
+
+</div>
+
+  <div id="content"></div>
+  <div id="loader"></div>
+   <div class="loaded"></div>
+    <%= render :partial => 'layouts/header' %>
+
+  </body>
+
+<script>
+var myVar;
+ function myFunction() {
+    myVar = setTimeout(showPage, 3000);
+}
+ function showPage() {
+  document.getElementById("Loader").style.display = "none";
```

Comment:
- document.getElementById() is case sensitive so this is returning null
- try `document.getElementById("loader").style.display = "none"`;